tools/generate-component/boilerplate/component.spec.ts

```diff
@@ -1,18 +1,25 @@
 import { expect } from '@open-wc/testing';
 import { html } from 'lit/static-html.js';
-import './__noPrefixName__';
-import { fixture } from '../../../src/elements/core/testing';
+
+import { fixture, testA11yTreeSnapshot } from '../core/testing/private.js';
+
+import type { __nameUpperCase__ } from './__noPrefixName__.js';
+import './__noPrefixName__.js';
 
 describe(`__name__`, () => {
-  it('renders', async () => {
-    const root = await fixture(html`<__name__ my-prop="Label"></__name__>`);
+  let element: __nameUpperCase__;
+
+  beforeEach(async () => {
```

Comment:
Can we nest this in another describe block having "renders" as title and then removing the duplicated title in "it"?